types/constants.go

```diff
@@ -103,21 +105,259 @@ func init() {
 		// that the network is secure in a real-world byzantine environment.
 		BlockFrequency = 600                     // 1 block per 10 minutes.
 		TargetWindow = 1e3                       // Number of blocks to use when calculating the target.
-		MaturityDelay = 50                       // 8 hours - 50 blocks.
+		MaturityDelay = 144                      // Certain payouts take 1 day to mature.
 		FutureThreshold = 3 * 60 * 60            // 3 hours.
 		ExtremeFutureThreshold = 5 * 60 * 60     // 5 hours.
-		GenesisTimestamp = Timestamp(1431000000) // 12:00pm UTC May 7th 2015
+		GenesisTimestamp = Timestamp(1433600000) // June 6th, 2015 @ 2:13pm UTC.
 
 		// A difficulty clamp to make long range attacks difficult. Quadrupling the
 		// difficulty will take 3000x work of finding a single block of the
 		// original difficulty. This can be compared to Bitcoin's clamp, in which
 		// quadrupling the difficulty takes 2000x work of finding a single block of
 		// the original difficulty.
-		MaxAdjustmentUp = big.NewRat(1001, 1000)
-		MaxAdjustmentDown = big.NewRat(999, 1000)
-		RootTarget = Target{0, 0, 0, 64}
+		MaxAdjustmentUp = big.NewRat(25, 10)
+		MaxAdjustmentDown = big.NewRat(10, 25)
+		RootTarget = Target{0, 0, 0, 4}
 		MinimumCoinbase = 30e3
 
 		RenterZeroConfDelay = 60 * time.Second
 	}
+
+	GenesisSiafundAllocation = []SiafundOutput{
+		{
+			Value:           NewCurrency64(2),
+			UnlockHash:      UnlockHash{4, 57, 229, 188, 127, 20, 204, 245, 211, 167, 232, 130, 208, 64, 146, 62, 69, 98, 81, 102, 221, 7, 123, 100, 70, 107, 199, 113, 121, 26, 198, 252},
+			ClaimUnlockHash: UnlockHash{4, 57, 229, 188, 127, 20, 204, 245, 211, 167, 232, 130, 208, 64, 146, 62, 69, 98, 81, 102, 221, 7, 123, 100, 70, 107, 199, 113, 121, 26, 198, 252},
```

Comment:
would be nice if we had a function that would convert a hex-encoded string to a Hash...